src/clients/Client.h

```diff
@@ -192,6 +192,8 @@ class Client:
     /// handleAdaptedBodyProductionEnded() was called
     bool receivedWholeAdaptedReply = false;
 #endif
+    bool receivedWholeVirginReply = false; ///< markParsedVirginReplyAsWhole() called
```

Comment:
I was thinking whether we need an optional<> here, but it looks like that a simple flag should be enough.